DeepSpeech.py

```diff
@@ -632,7 +670,7 @@ def create_inference_graph(batch_size=1, n_steps=16, tflite=False):
     }
 
     if not FLAGS.export_tflite:
-        inputs.update({'input_lengths': seq_length})
+        inputs['input_lengths'] = seq_length
```

Comment:
Why did we put `update` in the first place? I can't remember

Reply:
I don't know either. Maybe we were adding more than one key at once?